Sample-XML/Position Limits.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Snapshot Date="2017-01-03">
+  <Instruments>
+
+	<Future InstrumentID="11" InstrumentName="AGRICULTURE - BURSA MALAYSIA CRUDE PALM OIL GASOIL SPREAD (POG) FUTURES*" Market="CME" DeliveryMonth="2017-03" CommoditySymbol="POG" IsCashSettled="true"/>

```

Comment:
@Patrick-Willis I may be being stupid here. But do we use IsCashSettled anywhere in the rules atm? Im not aware of seeing anything that distinguishes in the limits between cash and physically settled